client/scss/components/_page_description.scss

```diff
@@ -12,6 +12,20 @@
   @include visually-hidden();
 }
 
+.page-description__intro {
+  display: block;
+
+  @include font('WB7');
+
+  @include respond-to('medium') {
+    @include font('WB6');
+  }
+
+  @include respond-to('large') {
+    @include font('WB5');
+  }
+}
```

Comment:
It does feel like we can abstract this type of font setting out, as we do it quite a lot...
Another PR perhaps.